### ISSUE TITLE
[accessibility] Use `<button>` or `<a role="button">`

### ISSUE THREAD

**Issue by Malvoz** [socib/Leaflet.TimeDimension#210]
Time controls are announced as a "link" using a screen reader (missing [`role="button"`](https://www.w3.org/TR/wai-aria/#button)).

Another option is to use `<button type="button">`, but which would require additional styling to reset browser defaults.

References:

- https://github.com/Leaflet/Leaflet/blob/bdd7ebf836201550f6d173943b728098b83f40d5/src/control/Control.Zoom.js#L89-L91
- https://www.w3.org/TR/2016/WD-wai-aria-practices-1.1-20160317/examples/button/button.html
